config/routes.rb

```diff
@@ -8,9 +8,11 @@
 
   post '/cookie/login', to: 'cookie_authentications#create'
   delete '/cookie/logout', to: 'cookie_authentications#destroy'
+  post '/cookie/logout', to: 'cookie_authentications#destroy'
   post '/cookie/refresh', to: 'refresh_cookies#create'
   post '/token/login', to: 'token_authentications#create'
   delete '/token/logout', to: 'token_authentications#destroy'
+  post '/token/logout', to: 'token_authentications#destroy'
```

Comment:
Sometimes it's just easier to have a `POST` do logout.  Simply adding it to the route-endpoints for convenience.